cypress/integration/transforms_spec.js

```diff
@@ -0,0 +1,247 @@
+        cy.get(`#_selection_column_${TRANSFORM_ID}-checkbox`)
+          .check({ force: true });
+
+        // Click on Actions popover menu
+        cy.get(`[data-test-subj="actionButton"]`).click({ force: true });
+
+        // Click Edit button
+        cy.get(`[data-test-subj="editButton"]`).click({ force: true });
+
+        // Wait for initial transform job to load
+        cy.contains("Test transform");
+
+        cy.get(`textArea[data-test-subj="description"]`).focus().clear().type("A new description");
+
+        // Click Save changes button
+        cy.get(`[data-test-subj="editTransformSaveButton"]`).click({ force: true });
+
+        // Confirm we get toaster saying changes saved
+        cy.contains(`Changes to transform saved`);
+
+        // Click into transform job details page
+        cy.get(`[data-test-subj="transformLink_${TRANSFORM_ID}"]`).click({ force: true });
+
+        // Confirm new description shows in details page
+        cy.contains("A new description");
+      });
+    });
+
+    describe("can be deleted", () => {
+      beforeEach(() => {
+        cy.createTransform(TRANSFORM_ID, sampleTransform);
+        cy.reload();
+      });
+
+      it("successfully", () => {
+        // Confirm we have our initial transform
+        cy.contains(TRANSFORM_ID);
+
+        // Disable transform
+        cy.get(`#_selection_column_${TRANSFORM_ID}-checkbox`).check({ force: true });
+        cy.get(`[data-test-subj="disableButton"]`).click({ force: true });
+        cy.contains(`"${TRANSFORM_ID}" is disabled`);
+
+        // Select checkbox for our transform job
+        cy.get(`#_selection_column_${TRANSFORM_ID}-checkbox`).check({ force: true });
+
+        // Click on Actions popover menu
+        cy.get(`[data-test-subj="actionButton"]`).click({ force: true });
+
+        // Click Delete button
+        cy.get(`[data-test-subj="deleteButton"]`).click({ force: true });
+
+        // Type "delete" to confirm deletion
+        cy.get(`input[placeholder="delete"]`).type("delete", { force: true });
+
+        // Click the delete confirmation button in modal
+        cy.get(`[data-test-subj="confirmModalConfirmButton"]`).click();
+
+        // Confirm we got deleted toaster
+        cy.contains(`"${TRANSFORM_ID}" successfully deleted`);
+
+        // Confirm showing empty loading state
+        cy.contains(
+          "Transform jobs help you create a materialized view on top of existing data."
+        );
+      });
+    });
+
+    describe("can be enabled and disabled", () => {
+      beforeEach(() => {
+        cy.createTransform(TRANSFORM_ID, sampleTransform);
+        cy.reload();
+      });
+
+      it("successfully", () => {
+        // Confirm we have our initial transform
+        cy.contains(TRANSFORM_ID);
+
+        // Click into transform job details page
+        cy.get(`[data-test-subj="transformLink_${TRANSFORM_ID}"]`).click({ force: true });
+
+        cy.contains(`${TRANSFORM_ID}`);
+
+        cy.wait(1000);
```

Comment:
What's this for

Reply:
Removed.

Reply:
Actually turns out that this one is required. It takes a moment for the data to load, and if we click into that menu, the retry happens after the menu is open, which doesn't re-render. A short wait alleviates this.